tests/InstrumentationEngine.ProfilerProxy.Tests/EventLoggerTests.cpp

```diff
@@ -120,40 +118,51 @@ namespace InstrumentationEngineProfilerProxyTests
             Assert::AreEqual(expectedCount, messageCount);
         }
 
-        // Tests EventLogger cannot write before initialization
-        TEST_METHOD(EventLoggerCannotWriteBeforeInitialize)
```

Comment:
>EventLoggerCannotWriteBeforeInitialize [](start = 20, length = 38)

Due to RAII (Resource Acquisition Is Initialization), it is impossible to get an uninitialized object since the constructor calls Initialize.